### DISCUSSION TITLE
Help with initial configuration, OTP problems

### DISCUSSION THREAD

**Discussion by FireWolfCL** [gravitational/teleport#39371] [Q&A]
Hi Teleport Community, 
I've installed my teleport server self-hosted, with self-signed certs. After that, I removed all configuration and installation to install again with a valid wildcard certificate for the site. However, my problem occurs when I try to configure my first User. Reading the QR with apps shows me the message "Token is invalid! / The Token you are attempting to add is invalid. Please alert your token provider."
How can I regenerate a new token for the admin account? Should I reinstall all again?
Best Regards,
Felipe.

**Comment by webvictim** [comment]
Did you delete the contents of the `/var/lib/teleport` directory before restarting with the ACME cert?

It shouldn't be strictly necessary, but if you're having issues and don't have any resources added to your cluster yet, it's an easy step to try.

Shut down Teleport, delete the contents of `/var/lib/teleport`, then start it up again and run `sudo tctl users add <admin-username> --roles access,editor,auditor` to generate a new user invite link.
